MaterialLibrary/src/main/java/io/doist/material/widget/RobotoEditText.java

```diff
@@ -3,11 +3,10 @@
 import android.content.Context;
 import android.graphics.Typeface;
 import android.util.AttributeSet;
-import android.widget.EditText;
 
 import io.doist.material.utils.Roboto;
 
-public class RobotoEditText extends EditText {
+public class RobotoEditText extends MaterialEditText {
```

Comment:
I think that `MaterialEditText` is the one that should extend from `RobotoEditText`.